companion/src/storage/firmwareinterface.cpp

```diff
@@ -256,6 +380,12 @@ bool FirmwareInterface::setSplash(const QImage & newsplash)
         b[idx] |= z;
       }
     }
+    if(splashWidth == 212 && splashHeight == 64)
+    {
+	    splashSize = RleBitmap::encode(b, SPLASH_SIZE_MAX);
+	    if(splashSize > 3072)
```

Comment:
Magic numbers 😬

Reply:
Yes and I do not like them.
Those, the max splash size define and the BltBitmap class are the reason for my question for a shared place for code and definitions between radio and companion

Reply:
found some defines and created a new one, but they are still not shared between radio and companion